### PR TITLE
Plugin status now shows externally configured services/images

### PR DESCRIPTION
**Pull request by cprafullchandra**

### CODE REVIEW

File: info_tools/get_status.py [hunk 1]
```diff
@@ -239,6 +244,14 @@ def get_installed_plugins(path_dirs):
 
     return p
 
+def get_installed_repos(path_dirs):
+    try:
+        return [ plugin_repo for plugin_repo in os.listdir(path_dirs.plugin_repos) if os.path.isdir(os.path.join(path_dirs.plugin_repos, plugin_repo)) ]
+    except Exception as e:
+        with open('/tmp/error.log', 'a+') as myfile:
+            myfile.write("Error - get_status.py: get_installed_repos")
+        pass
```
Comment (cglewis) [inline info_tools/get_status.py]: unnecessary `pass` ?

File: info_tools/get_status.py [hunk 2]
```diff
@@ -492,6 +505,30 @@ def get_enabled(path_dirs):
...
+        # check section exists
+        if config.has_section("external"):
+            # check if there are any options
+            if config.options("external"):
+                for option in config.options("external"):
+                    extconf[option] = config.get("external", option)
+    except Exception as e:
+        with open('/tmp/error.log', 'a+') as myfile:
+            myfile.write("Error - get_status.py: get_external")
+        pass
```
Comment (cglewis) [inline info_tools/get_status.py]: again.

Comment (cprafullchandra) [inline info_tools/get_status.py]: "THEY'RE EVERYWHERE." I'll clean them up! :)

Comment (cglewis) [inline info_tools/get_status.py]: lol, thanks.

**Pull request merged**